### PR TITLE
[#256] Update operator-sdk from v1.15.0 to v1.25.0 (go 1.19)

### LINKED ISSUE #256: Update operator-sdk from v1.15.0 to v1.28.0 (go 1.19)
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**
Optional. A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Optional. Add any other context or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by gaohoward**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,17 +41,28 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # This variable is used to construct full image tags for bundle and catalog images.
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# amq.io/e-sdk15-bundle:$VERSION and amq.io/e-sdk15-catalog:$VERSION.
+# aquay.io/artemiscloud/activemq-artemis-operator-bundle:$VERSION and quay.io/artemiscloud/activemq-artemis-operator-catalog:$VERSION.
```
Comment (brusdev) [inline Makefile:44]:
```suggestion
# quay.io/artemiscloud/activemq-artemis-operator-bundle:$VERSION and quay.io/artemiscloud/activemq-artemis-operator-catalog:$VERSION.
```

**Comment by gtully** [comment]
is the operator-sdk lock step with go versions?
Would it be possible to take on a later operator-sdk but stay at go1.17?

**Comment by gaohoward** [comment]
> 

That needs some evaluation.

**Comment by gaohoward** [comment]
nope, using go 1.17 doesn't compile. Some modules requires 1.18 and some 1.19.

**Comment by davidlanouette** [comment]
Can I ask what the status of this PR is?  It seems to have been stalled.

**Comment by brusdev** [comment]
This PR should be reviewed in the next 2~3 weeks

**Comment by davidlanouette** [comment]
Thank for the update.  I asked because I have been running into a lot of the same things that this PR fixes.  (My local dev env has a newer version of go and operator-sdk.)

**Comment by gaohoward** [comment]
close as it's superseded by https://github.com/artemiscloud/activemq-artemis-operator/pull/572

**Pull request closed without merging**